api/src/Microsoft.Azure.IIoT.Api/src/Publisher/Extensions/ModelExtensions.cs

```diff
@@ -776,12 +719,11 @@ public static class ModelExtensions {
                 return null;
             }
             return new DataSetWriterApiModel {
-                DataSetWriterId = model.DataSetWriterId,
+                DataSetWriterName = model.DataSetWriterName,
```

Comment:
Do we drop the support for DataSetWriterId with this?

Reply:
No, default is DataSetWriterId "string" in the encoder/decoder, only if --strict is specified it is compliant.  Also, models still read and write from the "dataSetWriterId" json property to maintain backcompat.